samples/rag-storage-document-loaders/query.py

```diff
@@ -0,0 +1,105 @@
+"""Sample demonstrating a chatbot using Azure AI and Azure Search."""
+
+import os
+import warnings
+
+from azure.identity import DefaultAzureCredential
+from dotenv import load_dotenv
+from langchain_azure_ai.chat_models import AzureAIChatCompletionsModel
+from langchain_azure_ai.embeddings import AzureAIEmbeddingsModel
+from langchain_azure_ai.vectorstores import AzureSearch
+from langchain_community.vectorstores.azuresearch import AzureSearchVectorStoreRetriever
+
+
+load_dotenv()
+warnings.filterwarnings("ignore", message=".*is currently in preview.*")
+
+
+def get_chat_model() -> AzureAIChatCompletionsModel:
+    """Initialize and return the Azure AI Chat Completions Model."""
+    chat_model = AzureAIChatCompletionsModel(
+        endpoint=os.environ["AZURE_CHAT_ENDPOINT"],
+        credential=DefaultAzureCredential(),
+        model="gpt-4.1-mini",
```

Comment:
For both the chat model and embeddings model, could we expose these as separate required env vars? Mainly it will allow a customer to use different models without changing the code.